backend/models/time.py

```diff
@@ -132,14 +132,12 @@ class OSRMDrivingTime(APIDrivingTime):
     Uses the `table` OSRM API endpoint.
     """
 
-    def __init__(self, api_url=None, early_exit_outer_radius=120.0 * 10**3):
+    def __init__(self, *args, **kwargs):
         """Initialize the distance class with the OSRM URL."""
         # FIXME: Once the API accepts distances, enable a `distance` mode.
         logger.debug('Initializing OSRM measurer.')
-        super().__init__()
-        self.api_url = api_url or os.getenv('OSRM_URL')
-        self._haversine_measurer = distance.HaversineDistance()
-        self.early_exit_outer_radius = early_exit_outer_radius
+        super(OSRMDrivingTime, self).__init__(*args, **kwargs)
+        self.api_url = self.api_url or os.getenv('OSRM_URL')
```

Comment:
What is `self.api_url` on the right-hand side? This is not set in the `APIDrivingTime.__init__` method.